docs/user_documentation.rst

```diff
@@ -93,6 +93,90 @@ respective roles below in :ref:`creating_projects_cli`,
 Using Commandline Interface
 ---------------------------
 
+Getting assistants
+~~~~~~~~~~~~~~~~~~
+
+By default DevAssistant comes with no assistants pre-installed and when you want some, you need
+to install them first from DAPI, DevAssistant Package Index, https://dapi.devassistant.org/.
```

Comment:
By default, DevAssistant comes with no Assistants pre-installed. If you want to install some, you can get them from DAPI....